wiki/Game_Modifiers/en.md

```diff
@@ -1,306 +1,608 @@
 
-Game modifiers can be accessed by pressing `F1` at song select, or by clicking the `Mods` button near the lower left of the screen.
+Shortcut keys to enable the mods using keyboard's keys can be changed at Options under Keyboard header inside `Change keyboard bindings` with a header called Mod Select. The usage of the shortcut keys will only work when the Mods screen is shown.
 
-Note that when multiple mods are applied, the product of the score multipliers is given instead of the sum of them. For example, when Hidden and Flashlight are applied at the same time, they give 1.1872x because (1.06 x 1.12) = 1.1872, rather than (1.06 + 1.12 - 1.00) = 1.18. *osu!* used to round this number down to the nearest hundredth (0.01), so Hidden and Flashlight together would appear to give a 1.18x multiplier, denying the supposed number round up. Thankfully, a patch corrected this, and now Hidden + Flashlight rounds correctly to 1.19x.
+The game modifiers screen can be accessed by pressing `F1` during song selection screen or by clicking the `Mods` button near the lower left of the screen.
 
-![Game mods screen](img/Soloplay8.png "Game mods screen")
+![Song Selection's Toolbox][Toolbox]
 
-Difficulty Reduction Mods
--------------------------
+## Difficulty Reduction
 
-These mods make the game easier at the cost of decreasing the score obtained.
+These mods can make the game easier at the cost of decreasing the score multiplier.
 
 ### Easy
 
-![Easy mod icon](/wiki/shared/Easy.png "Easy mod icon")
+![Easy (EZ) mod icon][EZ]
+
+> Score multiplier: 0.50
+
+> Shortcut key: `Q`
+
+Easy mod halves all difficulty settings of the beatmap selected.
+
+The Easy mod also grants the player two (2) additional "lives"; if the [life bar][lifebar wikilink] is exhausted completely, the play will pause and refills the life bar slowly to about 80% with no sound in-between at a cost of one "life" of the play. The refill is signaled by the ready sound (`readys.wav`) instead of the typical fail sound (`failsound.wav`) for failing the beatmap with an empty life bar. The beatmap will resume once the refill is successful signaled by playing the go sound (`gos.wav`). The play will fail instead if there is no "lives" remain when the life bar has been exhausted completely again.
+
+Interestingly, when watching a replay where extra "life" was used, the replay will skips over the health refill animation and allows the replay to continue at no health instead while allowing health regain from hit objects.
```

Comment:
draft of replacement:
`Interestingly, when watching a replay where an extra "life" was used, the replay will skip over the health refill animation and keep on going without any health instead while allowing health regain from hit objects.`